tla/issue.md

```diff
@@ -0,0 +1,150 @@
+# dBFT 2.0 formal modelling and the liveness problem investigation
+
+### Summary or problem description
+
+This issue is aimed to emphasize the dBFT 2.0 liveness lock problem mentioned in
+https://github.com/neo-project/neo/issues/2029#issuecomment-725612388 and in the
+set of other related issues, PRs and papers with the help of TLA+ formal
```

Comment:
mentioned in _link_ and other discussions in issues and PRs. We've used TLA+ formal modeling tool to analyze dBFT 2.0 behaviour.